smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/CommandGenerator.java

```diff
@@ -94,7 +94,14 @@ public void run() {
         addInputAndOutputTypes();
 
         String name = symbol.getName();
-        writer.openBlock("export class $L extends $$Command<$L, $L> {", "}", name, inputType, outputType, () -> {
+        writer.openBlock(
```

Comment:
Not everything for `openBlock` calls that go past the line length needs to be on its own line, [example](https://github.com/awslabs/smithy-typescript/blob/00622f9de152dddaa0f1e6921623ff215a8b84a4/smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/HttpBindingProtocolGenerator.java#L95). This would be fine:

```
writer.openBlock("export class $L extends $$Command<$L, $L> {", "}", name, inputType, outputType, configType,
                () -> {...
```

Similar consolidation can be done in a couple other places.